test/unit/filter-and-transform/filters/constains.spec.js

```diff
@@ -0,0 +1,56 @@
+const test = require('tape')
+const contains = require('../../../../lib/filter-and-transform/filters/contains')
+
+test('contains: empty input', t => {
+  const result = contains()
+  t.equals(result, false)
+  t.end()
+})
+
+test('contains: empty object input', t => {
+  const result = contains({}, {})
```

Comment:
Probably adding tests for `contains(null, {})` and `contains({}, null)` since you are validating the two parameters separately.